subprojects/core/src/main/java/org/gradle/api/internal/component/UsageContext.java

```diff
@@ -28,7 +28,8 @@
 import java.util.Set;
 
 public interface UsageContext extends HasAttributes, Named {
-    Usage getUsage();
+    @Deprecated
+    Usage getUsage(); // kept for backwards compatibility of plugins using internal APIs
```

Comment:
❓ If I understand correctly and given the implementation in the abstract class, that's purely for plugins that `implement` this interface, correct?

Reply:
That's correct. Several plugins are doing this.